tests/integration/ha/test_ha.py

```diff
@@ -430,3 +443,84 @@ async def test_scale_down_storage_re_use(ops_test: OpsTest, request):
 
     logger.info("Confirming writes replicated on new unit...")
     assert cw.count_znodes(parent=parent, hosts=new_host, username=USERNAME, password=password)
+
+
+@pytest.mark.abort_on_fail
+@pytest.mark.skip(
```

Comment:
It's probably better to mark this as `unstable` instead of `skip`, as the CI pipeline is already has the unstable mark. Also, it separates this test from others that are skipped because of bugs.